src/D2L.CodeStyle.Analyzers/D2L.CodeStyle.Analyzers.nuspec

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>$id$</id>
+    <version>$version$</version>
+    <title>$title$</title>
+    <authors>$author$</authors>
+    <owners>$author$</owners>
+
+    <projectUrl>https://github.com/Brightspace/D2L.CodeStyle</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>$description$</description>
+    <copyright>$copyright$</copyright>
+    
+    <frameworkAssemblies>
+      <frameworkAssembly assemblyName="System" targetFramework="" />
+    </frameworkAssemblies>
+
+    <dependencies>
+      <dependency id="D2L.CodeStyle.Analysis" version="$version$" />

```

Comment:
Does it need to be a different project? The naming suggests they are related.

And from the names, you need something to analyze (ie analyzers) before you can have an analysis.  So this sounds backwards